### PR TITLE
tox: Add ansible-lint env

### PR DESCRIPTION
**Pull request by acernek**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -94,6 +94,12 @@ commands =
 deps =
     -r docs/requirements.txt
 
+[testenv:ansible-lint]
+deps =
+    ansible-lint
+commands =
+    ansible-lint -x yaml[truthy],no-changed-when -p common/setup/setup_playbook.yml common/scripts/fetch_artifacts.yml
```
[the remark this answers is not in the document]
Comment (acernek) [reply tox.ini]: Done

File: common/setup/setup_playbook.yml [hunk 2]
```diff
@@ -94,11 +98,11 @@
               - "{{ python_pkg }}"
               - "{{ python_pkg }}-devel"
               - qemu-kvm
-            state: latest
+            state: present
```
[the remark this answers is not in the document]
Comment (acernek) [reply common/setup/setup_playbook.yml:101]: They don't specify their reasoning: "Package installs should not use latest. Package installs should use `state=present` with or without a version" [1]; I guess they want to force us to use specific versions.

[1] https://ansible-lint.readthedocs.io/en/latest/default_rules/#package-latest

[the remark this answers is not in the document]
Comment (tinez) [reply common/setup/setup_playbook.yml:101]: I think we should ignore this - this is a major change in logic.

[the remark this answers is not in the document]
Comment (acernek) [reply common/setup/setup_playbook.yml]: Done

**Review by tinez — APPROVED** [review]

**Pull request merged**